docs/guides/edge.md

```diff
@@ -67,3 +67,132 @@ with Diagram(name="Advanced Web Service with On-Premise (colored)", show=False):
 ```
 
 ![advanced web service with on-premise diagram colored](/img/advanced_web_service_with_on-premise_colored.png)
+
+## Less Edges
+
+As you can see on the previous graph the edges can become quickly noisy, here one example to solve this problem.
+
+One hack is to get creative with Custom to create blank placeholders, together with named nodes within Clusters, and then only pointing to single named elements within those Clusters.
```

Comment:
One ~~hack~~ **approach** is to get creative with ~~Custom~~ **the Node class** to create blank placeholders, together with named nodes within Clusters, and then only pointing to single named elements within those Clusters.